migrations/versions/0004_add_romm_repin_table.py

```
@@ -0,0 +1,57 @@
+"""add_romm_repin_table
+
+Revision ID: 0004
+Revises: 0003
+Create Date: 2026-08-17 22:14:37.943022
+"""
+
+from __future__ import annotations
+
+from typing import Sequence, Union
```

Ruff (0.16.1)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)